.github/FUNDING.yml

```diff
@@ -1,3 +1,3 @@
 # These are supported funding model platforms
 
-custom: https://nixos.org/nixos/foundation.html
```

Comment:
Is there any reason we should keep the foundation link too?

Reply:
I don't think so, because that will redirect users back to OC. They can always end up on foundation link from OC if they want to know the legal details.

Reply:
:+1:

Reply:
> Is there any reason we should keep the foundation link too?

are SEPA donations still supported? i would prefer it since it has no fees

If so, it makes sense to keep the link

Reply:
They are still supported, but they won't be transparent so the fees are for that.

Reply:
so can we keep the link and clarify this difference? also be transparent what you (the foundation) prefer.

Reply:
can't clarify the difference on girhub, but can do.on OC.

Reply:
i mean on OC and https://nixos.org/nixos/foundation.html

Reply:
> They are still supported, but they won't be transparent so the fees are for that.

@domenkozar it seems that it is possible to expose "external" donations: https://docs.opencollective.com/help/hosts/add-funds-manually

Reply:
Yes, but at this point it's not worth automating given the low yearly amounts.